### ISSUE TITLE
GETTFT v1.1.0

### ISSUE THREAD

**Issue by despiegk** [threefoldtech/home#1143]
- [ ] Add ‘Trade Fiat to TFT’ Integration

Project: https://github.com/threefoldfoundation/it_as_energy_shop/projects/9

On the first initial release of TFT Shop v1.0.0, we have made it easier for users to buy TFT by trading their BTC on our official [TFT Shop website](https://gettft.com/gettft/). 

On this v1.1.0 release, we aim to give the users another option to buy TFT by using fiat currency. This will be made possible by integrating [mercuryo.io](http://www.mercuryo.io/) (third-party) widget onto the shop. By buying TFT using TFT Shop you confirm that you read and agree to [ThreeFold’s terms and conditions.](https://library.threefold.me/info/legal/#/legal__terms_conditions_gettft)

In short:   
* No integration fees.  
* They support Visa, Mastercard, Google Pay, Apple Pay, IBAN. 

API docs:   
https://mercuryo.io/widget/   
https://github.com/mercuryoio/api-migration-docs/blob/master/Widget_API_Mercuryo_v1.6.md

github project:  
https://github.com/threefoldfoundation/it_as_energy_shop/projects/9

**Comment by xmonader** [comment]
Pending PR review https://github.com/threefoldfoundation/it_as_energy_shop/pull/173

**Comment by sasha-astiadi** [comment]
@xmonader can we put it to 'verification' then bc of the pr?

**Comment by robvanmieghem** [comment]
I'm on it

**Comment by robvanmieghem** [comment]
code complete, doing final verification
